testing/cli/test_cli_sct_label_utils.py

```diff
@@ -48,3 +48,39 @@ def test_create_seg_mid(tmp_path):
     # Old syntax for this behavior should not be allowed
     with pytest.raises(DeprecationWarning):
         sct_label_utils.main(['-i', input, '-create-seg', '-1,3', '-o', output])
+
+
+def test_project_centerline(tmp_path):
+    """Test the '-project-centerline' option in sct_label_utils"""
+    # Use an image as a shape reference
+    img = str(tmp_path/'t2.nii.gz')
+    Image(sct_test_path('t2', 't2.nii.gz')).change_orientation('RPI').save(img)
+
+    # Create a mask with a vertical line following the z axis with the coordinates x=20 and y=15
+    mask = str(tmp_path/'t2_seg.nii.gz')
+    sct_create_mask.main([
+         '-i', img,
+         '-o', mask,
+         '-size', '1',
+         '-p', 'coord,20x15',
+         '-f', 'box'
+         ])
+
+    # Create an image with a point at the coordinates x=1, y=1, and z=1 with the value=1
+    ref = str(tmp_path/'t2_seg_labeled.nii.gz')
+    sct_label_utils.main([
+        '-i', img,
+        '-o', ref,
+        '-create', '1,1,1,1'
```

Comment:
It would be nice to use different values for the coordinates, for example `1,2,3,4`, so that the test has a chance to detect more possible errors, like swapped coordinates or something.

Reply:
I added 2 other points